apps/docs/content/components/layout/Flex.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flex (WIP)
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/f8995017-240d-48bf-9877-98f92cde023c)
I'd try to add more padding